docker/Dockerfile.tests

```diff
@@ -0,0 +1,135 @@
+FROM phusion/baseimage:0.11
```

Comment:
This image was built locally and pushed to docker hub as `metabrainz/musicbrainz-tests:v-2019-02`.

It's intended only as a base image: we only have to build it (and push a new version) when we want to update software that the tests use, like node, chrome, postgres, etc. It doesn't include any code in the image; all code is checked out/built/cached on CircleCI.

Reply:
Also, having postgres, redis, and musicbrainz-server in the same container vastly simplified the setup and configuration. Obviously we wouldn't do that in production, but I think it's perfectly fine for the tests.